ui/pages/routes/routes.component.js

```diff
@@ -635,11 +648,162 @@ export default class Routes extends Component {
     }
   };
 
-  render() {
+  renderToasts() {
+    const { t } = this.context;
     const {
       account,
       activeTabOrigin,
+      addPermittedAccount,
+      showSurveyToast,
       showConnectAccountToast,
+      showPrivacyPolicyToast,
+      newPrivacyPolicyToastShownDate,
+      clearSwitchedNetworkDetails,
+      setSurveyLinkLastClickedOrClosed,
+      setNewPrivacyPolicyToastClickedOrClosed,
+      setSwitchedNetworkNeverShowMessage,
+      switchedNetworkDetails,
+    } = this.props;
+
+    const showAutoNetworkSwitchToast = this.getShowAutoNetworkSwitchTest();
+    const isPrivacyToastRecent = this.getIsPrivacyToastRecent();
+    const isPrivacyToastNotShown = !newPrivacyPolicyToastShownDate;
+
+    return (
+      <ToastContainer>
```

Comment:
I suggest moving this function into its own component and then importing it. This approach offers several benefits: it makes the function easier to test on its own (unit testing), enables us to create a dedicated Storybook entry for it, and facilitates its reuse in future projects or components if needed, and reduce the size of the` routes.component.js` file